model/flow/service_event.go

```diff
@@ -10,10 +10,16 @@ import (
 	cborcodec "github.com/onflow/flow-go/model/encoding/cbor"
 )
 
+type ServiceEventType string
+
+func (set ServiceEventType) String() string {
```

Comment:
maybe add a todo to clean up the plumbing.  general event's Type_ should probably just be EventType instead of a string